### ISSUE TITLE
Lyrics API

### ISSUE THREAD

**Issue by luxtiasco** [trifleneurotic/EmbySub#3]
Will you consider integrating lyrics API as Emby has it as its new feature in ver 4.7.
Related subsonic api wiki: http://www.subsonic.org/pages/api.jsp#getLyrics

**Comment by trifleneurotic** [comment]
Yes absolutely. Since it looks as if the Subsonic API has supported lyrics since 1.2.0 anyway, that will be a target for the first release. Will leave this issue open and will keep it updated as work progresses.

**Comment by trifleneurotic** [comment]
@luxtiasco 

Just wanted to let you know I'm getting back to his project after a long absence (four kids & various life events). This is still on my radar and I will be sure to update when I've made some progress.

**Comment by luxtiasco** [comment]
Thanks a lot!

**Comment by trifleneurotic** [comment]
@luxtiasco 

Hello! Just an FYI - I've pushed up the scaffolding for the API method group dealing with media retrieval, including lyrics, so it'll be in the first "official" release :) Appreciate your patience.
